docs/releases/5.2.txt

```diff
@@ -70,7 +70,10 @@ Minor features
 :mod:`django.contrib.postgres`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-* ...
+* :class:`~django.contrib.postgres.fields.SerialField`,
+  :class:`~django.contrib.postgres.fields.BigSerialField` and
+  :class:`~django.contrib.postgres.fields.SmallSerialField` have been added
+  corresponding to their respective PostgreSQL database types.
```

Comment:
@nessita do you think we should promote this a headline feature?

(nit: we will change this to present tense but want to check where we should put it first)

Reply:
On reflection, let's leave it for now. I know you have bigger things planned and we can always promote it later 👍